Contributing/How-To-Continuous-Integration.md

```diff
@@ -1,11 +1,12 @@
 ## Continuous Integration Tools
 
-While not yet implemented, moja global intends to utilise Continuous Integration Tools.
```

Comment:
You could use some of the FLINT lifecyle analysis to describe why continuous integration is a good thing. It fosters innovation by ensuring that ongoing improvements are always available and compatible to a set of users. This would allow a user developing their own countries implementation could progressively add new data and features while maintaining a functioning, reproducible system: https://docs.google.com/document/d/1vNSvKFERxvgUjEpBcG18dGaHhNnBC7hdPkkkcKBkeic/edit

Reply:
Done! Thanks for this content.